packages/react-charting/src/components/DonutChart/Arc/Arc.types.ts

```diff
@@ -139,6 +139,16 @@ export interface IArcStyles {
    */
   focusRing: IStyle;
 
+  /**
+   * Style for tool tip
+   */
+  tooltip?: IStyle;
```

Comment:
Are these properties getting used in Arc component.

Reply:
removed as it had moved to donut chart base